roles/move_objects_between_buckets/README.md

```diff
@@ -0,0 +1,112 @@
+move_objects_between_buckets
+==================
+
+A role to move objects from one S3 Bucket to another.
+Objects have two options: all objects can be transferred or specific objects are transferred through key prefix.
+If the source bucket is empty, the user has two options: source bucket is deleted or source bucket is kept as an empty bucket.
+
+Requirements
+------------
+
+AWS User Account with the following permission:
+
+* s3:HeadBucket
+* s3:GetBucketOwnershipControls
+* s3:ListObjectsV2
+* s3:ListBucket
+* s3:DeleteObject
+* s3:HeadObject
+* s3:PutObjectAcl
+* s3:CopyObject
+* s3:GetObjectTagging
+* s3:DeleteBucket
+
+Role Variables
+--------------
+
+* **source_bucket**: The name of the Amazon S3 bucket that will have its objects retrieved and then emptied. **Required**
+* **dest_bucket**: The name of the Amazon S3 bucket that will receive the objects. **Required**
+* **key_prefix**: limits objects that begin with the specified prefix. Default value is **""**.
+* **delete_empty_source_bucket**: deletes source bucket after all objects have been transferred to destination bucket. Default value is **false**.
+
+Dependencies
+------------
+
+- role: [aws_setup_credentials](../aws_setup_credentials/README.md)
+
+## Example:
+```
+---
+- name: Playbook for move objects between buckets using cloud.aws_ops.move_objects_between_buckets role
+  hosts: localhost
+  gather_facts: false
+  tasks:
+    - name: Define bucket name
```

Comment:
also I think we don;t need to provide entire playbook for example
probably just the `role` usage itself would be enough, so the tasks that use the role are good enough

for example refer https://github.com/redhat-cop/cloud.aws_ops/blob/main/roles/manage_vpc_peering/README.md#example